### PR TITLE
feat-add an independent page for sponsors

### PR DESCRIPTION
**Pull request by SivanYeh**
* **New feature**

## Related Issue

#426

### CODE REVIEW

**Comment by SivanYeh** [comment]
@josix is it the right way to achieve this feature?

File: pages/about/sponsor.vue [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+import SponsorCard from '@/components/sponsors/SponsorCard'
+import SponsorCardCollection from '@/components/sponsors/SponsorCardCollection'
+
+export default {
+    layout(context) {
+        const ccip = context.query.ccip // to determine if it's opass mobile app
+        if (ccip) {
+            return 'ccip'
+        }
+        return 'default'
```
Comment (mattwang44) [inline pages/about/sponsor.vue:57]: 前一個 ccip layout switch PR 的同個地方是否改了什麼XD

Comment (SivanYeh) [reply pages/about/sponsor.vue:57]: Thanks for notice lol

**Comment by josix** [comment]
@SivanYeh I've created another PR #452 to fix the issues that couldn't display the sponsors' logos. PTAL thanks!

**Pull request closed without merging**